web/.eslintrc.json

```diff
@@ -17,6 +17,7 @@
         }
       }
     ],
+    "no-param-reassign": [2, { "props": false }],
```

Comment:
This allows us to update the props of a param, which feels to me like the 90% case.  This is an older rule meant to prevent people from modifying the `arguments` object, which we don't use (and heavens, nobody should anymore).

Anyway, now we can do things like this:

```js
arr.forEach(o => { o.value += 3; });
```

Reply:
👍